strings/revenant_names.json

```diff
@@ -0,0 +1,49 @@
+{
+    "spirit_type": [
+        "Spirit",
+        "Ghost",
+        "Spectre",
+        "Phantom",
+        "Revenant",
+        "Essence",
+        "Soul"
+    ],
+
+    "adverb": [
+	"",
+        "abysmal ",
+        "dark ",
+        "eternal ",
+        "endless ",
+        "searing ",
+        "abyssal ",
+        "vicious ",
+        "hateful ",
+	"hideous ",
+	"malevolent ",
+	"fiery ",
+	"silent ",
+	"violent ",
+	"peaceful ",
+	"indifferent ",
+	"brutal "
+    ],
+
+    "theme": [
+        "despair",
+        "agony",
+        "screams",
+        "vengeance",
+	"hellfire",
+	"anger",
+	"suffering",
+	"death",
+	"destruction",
+	"hate",
+	"misery",
+	"disease",
+	"desperation",
+	"sorrow",
+	"ruin"
+    ]
```

Comment:
Add salt

Reply:
revenents are annihilated by salt which makes their ability to attack players weird